assets/scss/_single-connector.scss

```diff
@@ -1,5 +1,6 @@
 .single-connector {
   &__nav {
+    background: #0059d0;
```

Comment:
maybe we can turn this into a variable?

Reply:
Yeah, great call Malcolm! We actually have this already as a variable. I'll fix it :)